docs/source/markdown/podman-run.1.md

```diff
@@ -1011,8 +1011,14 @@ When set to **true**, Podman will allocate a pseudo-tty and attach to the standa
 input of the container. This can be used, for example, to run a throwaway
 interactive shell. The default is **false**.
 
-**NOTE**: The **-t** option is incompatible with a redirection of the Podman client
-standard input.
+**NOTE**: The **-t** option is incompatible with a redirection of the Podman client standard input.
+
+```
+Use
+echo "asdf" | podman run --rm -i someimage /bin/cat
+Versus
+echo "asdf" | podman run --rm -it someimage /bin/cat
```

Comment:
Both sample commands have equal visual prominence, making it hard for my eye to figure out which is the correct one. I'm a fan of "show what to do, and hide the taboos". Could I suggest something like:

...incompatible with a redirection of the Podman client standard input. That is, when feeding input to podman, use `-i` only, not `-it`:
```
    $ echo asdf | podman run --rm -i someimage /bin/cat
```